examples/guessing_game.py

```diff
@@ -10,17 +10,24 @@
 async def guess(ctx):
     await ctx.send("Guess a number between 1 and 10.")
 
-    def is_correct(m):
-        return m.author == ctx.author and m.content.isdigit()
 
-    answer = random.randint(1, 10)
-
-    try:
-        guess = await bot.wait_for("message", check=is_correct, timeout=5.0)
-    except asyncio.TimeoutError:
-        return await ctx.send(f"Sorry, you took too long it was {answer}.")
-
-    if int(guess.content) == answer:
+@bot.command()
+async def guess(ctx, guess: int = None):
+    def is_correct(message):
+        return message.author == ctx.author and message.content.isdigit()
+
+    answer = random.randin(1, 10)
+
+    if not guess:
+        await ctx.send("Guess a number between 1 and 10.")
+```#my bad
```

Comment:
```suggestion
        await ctx.send("Guess a number between 1 and 10.")
```
what the fuck github